### PR TITLE
Add-email

### PR DESCRIPTION
**Pull request by ehanson8**
### How can a reviewer manually see the effects of these changes?
Testing must occur on stage for proper access to the Data Warehouse and the tester must be a part of the `lib-alma-notifications` list:

- Export stage credentials and set `ECR_NAME_STAGE` and `ECR_URL_STAGE` env variables for `alma-patronload-stage`.
- To build and publish the container to stage, run `make dist-stage` and `make publish-stage`.
- Run the ECS task with output from TF Cloud for the stage environment.
- Observe that 2 files are created in the stage-sftp-shared bucket in S3 and that an email is sent to `lib-alma-notifications`

### Includes new or updated dependencies?
YES

### What are the relevant tickets?
NA

### Developer
- [x] All new ENV is documented in README (or there is none)
- [x] Stakeholder approval has been confirmed (or is not needed)

### Code Reviewer

- [x] The commit message is clear and follows our guidelines
      (not just this pull request message)
- [x] There are appropriate tests covering any new functionality
- [x] The documentation has been updated or is unnecessary
- [x] The changes have been verified
- [x] New dependencies are appropriate or there were no changes

### CODE REVIEW

File: patronload/email.py [hunk 1]
```diff
@@ -0,0 +1,80 @@
+from email.message import EmailMessage
+from email.policy import EmailPolicy, default
+from typing import List, Optional
+
+import boto3
+
+
+class Email(EmailMessage):
+    """Email sublcasses EmailMessage with added functionality to populate and send."""
```
[the remark this answers is not in the document]
Comment (ehanson8) [reply patronload/email.py:9]: Good catch, I'm also updating `alma-creditcardslips` since this is a module copy-paste

File: patronload/cli.py [hunk 2]
```diff
@@ -33,8 +35,8 @@ def main() -> None:
     config_values = load_config_values()
     root_logger = logging.getLogger()
     logger.info(configure_logger(root_logger, os.getenv("LOG_LEVEL", "INFO")))
+    stream = create_log_stream_for_email(root_logger)
```
Comment (adamshire123) [inline patronload/cli.py:38]: question just so I can understand better... Creating this log stream doesn't prevent logs from going to cloud watch, it just adds an additional place where log messages end up, is that right?

Comment (ehanson8) [reply patronload/cli.py:38]: Correct, this is an additional handler and everything still outputs to CloudWatch as expected as you'll see when running it manually

**Review by adamshire123 — APPROVED** [review]
👍

**Pull request merged**